### PR TITLE
Adding new factory and tests

### PR DESCRIPTION
**Pull request by apmilen**
I couldn't decide if I should combine these two multisig factories into one

### CODE REVIEW

File: contracts/multisig_factory.sol [hunk 1]
```diff
@@ -1,9 +1,20 @@
...
     function createBasicMultisig(
         address[] members, uint8 quorum, uint40 window
-    ) returns (DSBasicMultisig) {
-        return new DSBasicMultisig(members, quorum, window);
+    ) returns (DSBasicMultisig ret) {
+        ret = new DSBasicMultisig(members, quorum, window);
+        is_basic_multisig[address(ret)] = true;
+    }
+
+    function isBasicMultisig(address code)
```
Comment (dbrock) [inline contracts/multisig_factory.sol]: Better to declare the variable like this:

```
mapping (address => bool) public isBasicMultisig;
```

Then you can omit the function as you will get it for free.

**Comment by dbrock** [comment]
I think it probably makes more sense to combine them into one.

**Comment by apmilen** [comment]
the reason I wanted to make them separate is that it makes it easier to deprecate EasyMultisig

**Comment by dbrock** [comment]
Got it... probably doesn't make much of a difference. I don't see any problem with having them separate. Maybe the other one should be called `basic_multisig_factory.sol`?

**Pull request merged**